config/initializers/git.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+min = "2.5.0"
```

Comment:
Constant?

Reply:
not reused, so prefer not adding anything to the global namespace